bin/lib.sh

```diff
@@ -0,0 +1,20 @@
+if [ -n "$OPENSHIFT_HOMEDIR" ]; then
```

Comment:
This script is only meant to be sourced, executing it wouldn't do anything, it doesn't have the executable bit so a shebang wouldn't do anything here.